### DISCUSSION TITLE
support for multi tenant architecture

### DISCUSSION THREAD

**Discussion by himeshp** [Etheonor/supanextail#27] [Feedback]
https://github.com/supabase/supabase/discussions/1615

discussion here.

**Comment by Etheonor** [comment]
What is the idea behind that? Having several admin for example?

I'm wokring on a admin dashboard and the idea is to have an admin list in the database with a simple boolean true/false for each user. That way the owner will have the possibility to grant admin access to other persons.

**Reply by himeshp** [reply]
basically, I have multiple organizations and user management for each org.
see this https://aws.amazon.com/blogs/database/multi-tenant-data-isolation-with-postgresql-row-level-security/
